### DISCUSSION TITLE
Youtube RSS feed only grabs the las 15 video posts.

### DISCUSSION THREAD

**Discussion by EmQsed** [FreshRSS/FreshRSS#6511] [Q&A] [answered]
HI ! I tried with multiple channels and with the YouTube extension for FreshRSS yet I can only see the last 15 posted videos and nothing older. Is this expected behavior ? Is there a way to grab the whole history of a channel as a feed ?
Thanks !

**Comment by Alkarex** [comment, marked as answer]
15 is the typical number of entries in the RSS feeds returned by YouTube, so you cannot easily get older posts (besides doing a custom scraper).
However, after you have subscribed to a YouTube RSS feed, you will build up a longer history of all the newer posts.

**Reply by Frenzie** [reply]
RSS-Bridge seems to have implemented some custom JSON extraction to surpass 15, but only for playlists and not for pre-existing feeds. Perhaps it would be relatively straightforward to expand, provided regular video listings use a similar JSON.
https://github.com/RSS-Bridge/rss-bridge/blob/5a68ee0c87bc2392886efd411511e094ae4fe5ac/bridges/YoutubeBridge.php
